packages/info/configParser.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from "path";
+import chalk from "chalk";
+import * as prettyjson from "prettyjson";
+
+export function getNameFromPath(fullPath: string): string {
+	const filename = fullPath.replace(/^.*[\\\/]/, "");
+	return filename;
+}
+
+export function resolveFilePath(relativeFilePath: string) {
+	const configPath = path.resolve(process.cwd() + "/" + relativeFilePath);
+	return configPath;
+}
+
+export function fetchConfig(configPath: string) {
+	let config = null;
+	config = require(configPath);
+	try {
```

Comment:
what is happening here? why empty try? Do you mean to put config inside the try block?

Reply:
Good catch, actually while debugging I shifted the require statement out of the `try`.

After making require the child of `try`. (When module related err occurs)
![Screenshot from 2019-06-12 00-46-39](https://user-images.githubusercontent.com/32242596/59300003-aeb67480-8cab-11e9-99f5-76bf526def82.png)